roles/sap_ha_prepare_pacemaker/tasks/preconfig.yml

```diff
@@ -0,0 +1,6 @@
+---
+# [A] Change hacluster password
+- name: "SAP Prepare Pacemaker - Ensure password for hacluster is configured"
+  ansible.builtin.user:
+    name: hacluster
+    password: "{{ sap_ha_prepare_pacemaker_hacluster_password | password_hash('sha512') }}"
```

Comment:
The password_hash encrypts the provided pw during Ansible runtime so that it cannot be displayed plain-text at any stage. The user module's password parameter requires this. The password is provided through the "...hacluster_password" variable.